### PR TITLE
tarsnap-gui 0.9

### PR DESCRIPTION
**Pull request by shinnok**
Bumped version and added macdeployqt post-build step.

### CODE REVIEW

File: tarsnap-gui.rb [hunk 1]
```diff
@@ -18,6 +18,7 @@ class TarsnapGui < Formula
   def install
     system "qmake"
     system "make"
+    system "macdeployqt Tarsnap.app"
```
Comment (dunn) [inline tarsnap-gui.rb]: These should be two arguments to `system`:
```ruby
system "macdeployqt", "Tarsnap.app"

Comment (DomT4) [inline tarsnap-gui.rb]: 👍 LGTM aside this issue.

Comment (shinnok) [inline tarsnap-gui.rb]: Fixed, thanks for the quick feedback!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-gui/commit/ab794f527c41235ffea70c00f809c1b3969e07ea. Thank you for your contribution to Homebrew @shinnok; we appreciate it! :smiley_cat:

**Pull request closed without merging**